### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by mnlipp** [TheElectronWill/night-config#143] [General]
When will there be a next release? I appreciate the effort that you have put voluntarily into the library and I know it is hard to maintain such a project. But I assume that you also want it to be used by others.

I've reported [a bug](https://github.com/TheElectronWill/night-config/issues/121) in October 2022, which was fixed in January. That's okay, I could have provided a PR. The problem is that there have been no releases since August 2022. So although the bug has been fixed, things still fail for me (I used TOML as workaround then, but this is no longer possible).

I've looked through the issues and almost all open issues are classified as enhancements. Wouldn't it be possible to provide bug fix release more frequently? Just asking...

**Comment by TheElectronWill** [comment]
Hello!
> I've looked through the issues and almost all open issues are classified as enhancements. Wouldn't it be possible to provide bug fix release more frequently? Just asking...

Yes, definitely possible, sorry for not releasing one sooner, it seems like I just forgot 🙃 Of course, the goal is to make the new versions available as soon as possible, but sometimes life gets in the way...
I'll be able to publish one when I get access to my laptop, in a few days (it's in a cardboard box right now).

For now, you can use **jitpack** to add the last unreleased version as a dependency of your project.
=> https://jitpack.io/#TheElectronWill/night-config/master-SNAPSHOT

**Reply by mnlipp** [reply]
> I'll be able to publish one when I get access to my laptop, in a few days (it's in a cardboard box right now).

Thanks! Looking forward to it.

**Reply by TheElectronWill** [reply]
I'm on it :) will finish tomorrow (had to reconfigure gpg)

**Reply by TheElectronWill** [reply]
new release published, it will be available soon on maven central
